src/chat.rs

```diff
@@ -926,6 +937,44 @@ impl Chat {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub enum ArchiveState {
+    /// Neither archived or pinned
+    Normal = 0,
```

Comment:
If you don't use `#repr[u32]` you don't need to assign these numbers

Reply:
the numbers are also used in sqlite and also from the ffi, so it might be good to be explicit here.

Reply:
There is no relation to the ffi though, it's already bad enough that in the ffi code duplicates these numbers with the deltachat.h file, and now here is another place that carries the numbers, but it doesn't do anything here.  I think that's confusing and would rather not have that.

For the sql my argument is more or less similar, the only thing that matters there is that ToSql/FromSql agree with each other.  Both ignore these numbers here so just another place that will get out of date and confusing.

Reply:
you are right, i now understand :) i will change that.